sys/conf/kern.opts.mk

```diff
@@ -79,10 +79,6 @@ BROKEN_OPTIONS+= ZFS SSP
 BROKEN_OPTIONS+= ZFS
 .endif
 
-.if ${MACHINE_CPUARCH} == "riscv"
```

Comment:
Much rejoicing.  This bit me working on VM_PROT_CAP as RISC-V built fine but only MIPS found missing arguments in assertions.